stl/inc/format

```diff
@@ -1248,15 +1243,426 @@ _OutputIt _Write(_OutputIt _Out, basic_string_view<_CharT> _Value) {
     return _STD copy(_Value.begin(), _Value.end(), _Out);
 }
 
-// TODO: placeholder write
-template <class _CharT, class _OutputIt, class _Ty>
-_OutputIt _Write(_OutputIt _Out, _Ty _Val, _Basic_format_specs<_CharT>& _Specs) {
+template <class _CharT, class _OutputIt, class _Func>
+_OutputIt _Write_aligned(
+    _OutputIt _Out, int _Width, _Basic_format_specs<_CharT>& _Specs, _Align _Default_align, _Func&& _Fn) {

```

Comment:
I assume `_Width` could be const. Is there a reason this is `int`

Reply:
Is this a style that we follow? I personally find non-signature affecting `const` overly verbose, but if it's something done in the STL I can change it. I used `int` because `_Specs._Width` is also an integer, which makes subsequent comparisons/subtractions nice.

Reply:
It's something we usually like to do. I very frequently forget it as well.

Reply:
I am ok with that, but then we need to change all the writers, since none of them actually change their values. I think a _lot_ of parameters an become const, but it would make this changelog a lot longer. Would you be opposed to me auditing it in another PR?

Reply:
I've added a To Do card to the Text Formatting Project so this can be cleaned up later without slowing down this PR.

(It is somewhat verbose, and our codebase is inconsistent especially in older code, but attempting to follow it in new code helps avoid mistaken assignments, and makes it clearer when we do intend to modify a value parameter.)